src/autoscaler/testhelpers/ginkgo_extentions.go

```diff
@@ -8,6 +8,6 @@ import (
 
 func FailOnError(message string, err error) {
 	if err != nil {
-		ginkgo.Fail(fmt.Sprintf("%s: %s", message, err.Error()))
+		ginkgo.Fail(fmt.Sprintf("%s: %s", message, err.Error()), 1)
```

Comment:
Could we give this magic number a name?

Reply:
Im going to merge this in due to the nature of ginkgo you should be able to work it out.